packages/ui/src/Editor.tsx

```diff
@@ -21,6 +22,7 @@ export function Editor() {
         />
       }
       end={<Timeline />}
+      footer={<Footer />}
```

Comment:
`ResizeableLayout` is meant to have only two panes.
We should put the status bar under the timeline instead:
```suggestion
      end={
        <div className="add-necessary-css">
          <Timeline />
          <Footer />
        </div>
      }
```